tutorials/introductory/getting_started.py

```diff
@@ -0,0 +1,767 @@
+#
+# Two Approaches for Creating Graphs
+# ----------------------------------
+#
+# The two strategies, explicit and implicit, both involve using the ``pyplot``
+# module. However, they differ in how users interact with the data in the
+# transformation process. The `Introduction`_ and `Examples`_ sections above
+# provide key differences.
+#
+# +------------------------------------+------------------------------------+
+# | Explicit                           | Implicit                           |
+# +====================================+====================================+
+# | * Code has explicit references to  | * The programming is designed to   |
+# |   objects. Users manage objects for|   remember preceding events or     |
+# |   the specific Figure and Axes and |   interactions. This preserved     |
+# |   call on methods for manipulating |   state allows Matplotlib to       |
+# |   data.                            |   automatically manage a Figure and|
+# | * Object Oriented Programming      |   Axes.                            |
+# |   allows for robust control and is | * The module ``pyplot`` operates   |
+# |   useful for generalized code.     |   similarly to MATLAB and is       |
+# |                                    |   convenient for interactive       |
+# |                                    |   environments.                    |
+# +------------------------------------+------------------------------------+
+#
+# .. note::
+#
+#     The Matplotlib community does not recommend interchanging explicit and
+#     implicit strategies. When using one as standard, all code should follow
+#     the same strategy. Switching back and forth between explicit and
+#     implicit programming may yield errors.
+#
+# For other techniques of creating plots with Matplotlib, refer to
+# :ref:`user_interfaces`.
+#
+# Data
+# ----
+#
+# The Matplotlib library manages data in the form of iterables and/or
+# sequenced items. These can also take the form of NumPy arrays like
+# `numpy.array` or `numpy.ma.masked_array`. All plotting functions take these
+# data structures.
+#
+
+# Sample Data for Personal Financial Tracking in 2009 & 2010
+
+months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun',
+          'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
+income = [950, 950, 950, 950, 950, 950,
+          950, 950, 950, 950, 950, 950]
+chk_acct_09 = [1250, 1325, 1200, 1220, 1100, 1055,
+               1255, 1090, 1190, 1205, 1205, 1180]
+svg_acct_09 = [1000, 1050, 1100, 1150, 1200, 1250,
+               1300, 1350, 1400, 1450, 1500, 1550]
+chk_acct_10 = [1180, 1270, 1280, 1280, 1260, 1140,
+               1270, 1160, 1120, 1250, 1270, 1160]
+svg_acct_10 = [1550, 1600, 1650, 1700, 1750, 1800,
+               1850, 1900, 1950, 2000, 2050, 2100]
+
+##############################################################################
+#
+# .. note::
+#
+#    Other containers, such as `pandas` data objects, may not work as
+#    intended.
```

Comment:
not sure if this specifically is true so would maybe tilt back to being vague (Not all containers will work as intended)

Reply:
I think it's better for warning-like language not to be too broad with future tense as it may seem like a version of, _"There's something definitely broken, but we don't have the space here to tell you."_ I can remove the specific `pandas` reference though, if that no longer is true or if a programmer's expectations are different.

Reply:
Yeah for the most part pandas dataframes should just work/work with the data kwarg.